### ISSUE TITLE
More autogenerated files in Doc/Bib

### ISSUE THREAD

**Issue by olexandr-konovalov** [gap-system/GapWWW#185]
`Doc/Bib/Year/year.html` and `Doc/Bib/MSC/msc.html` can be generated automatically from `convbib.g` together with other files from `Doc/Bib/Year/` and `Doc/Bib/MSC`.

**Comment by fingolfin** [comment]
Actually, you could simply generate a single big `_data/bib.yml` file, containing an array of dictionaries; each dictionary is one bibitem. Something like this:
```yaml
- key: AAG19
  MSC: [05, 13, 14]
  mathscinet: 3942387
  author: "Abbas, A., Assi, A., and Garc\'ia-S\'anchez, P. A."
  title: "Canonical bases of modules over one dimensional $\boldK$-algebras"
  journal: "Rev. R. Acad. Cienc. Exactas F\'is. Nat. Ser. A Mat. RACSAM"
  volume: 113
  number: 2
  year: 2019
  pages: 1121–1139
- key: AJ19
...
```
Then we can generate the content of all Year and MSC files from that.

**Comment by fingolfin** [comment]
Meaning: we can write some jekyll templates that generate those files on the fly; and the formatting of bib entries could then be adjusted without re-running the tool that produces that YAML file.

**Comment by alex-konovalov** [comment]
Thanks - while we still want to keep bib entries too, I see benefits of adding `_data/bib.yml` too.

**Comment by fingolfin** [comment]
I don't understand what you mean with "keep bib entries, too" ? Are you talking about the bibtex record? But these can also be regenerated from the data in `_data/bib.yml`...

**Comment by fingolfin** [comment]
Resolved by PR #339 which did the new site design and dropped most of our bib stuff
